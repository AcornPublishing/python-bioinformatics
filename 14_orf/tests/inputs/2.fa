>Rosalind_2116
CGTGTTGAAGCACCAGATCGTGGGTCCCCTCGCCACCAACAGTCTGAAGTAGGTTGCTGC
TCTTAGCCAAAGACTGGGGGTTCGAGTCACGCCTGCAATTGTTACCACTTCGGCAGACAC
CGCACGTATTGACGGGGGTATGCCATGCTATGGACAAACACAAGAAATGGACGCCCGTAT
TTCTACTACCTTTGAGCTGCCCTATCTCACTATCTGTCAGGCATTTACATTCTGATGGCT
GAGGGGGGTTATCGTACTTCCAATCACGGGTCGTCCCTCTGATTCTCCTTGCGGCTAATC
CGCACACTCTATGACGACCCAAAAAGCTGGGATACAATGGGCTTACAATAAGTTGTTGAC
AACTGCCTTTGGAGTAACTAAGCTAGATTAAGAAATTCCTGGTAGCAGTCAATAACGCAA
GCTAGAATAAAAATACCCTCACTCTAAGCCGACCCTCTCTGTAATGTCAGAGGAAACGGA
CTAGCTAGTCCGTTTCCTCTGACATGCGCTTATCCAGAATCGCACTCAACCGTTTCCCCA
CTGGACAAAGAAACCGTCTGGCATACGACAAGTAACCTTGCAAAAGCAGGCCGGCGACGT
TCTTGCTGAGTGAGGAGAAACCGTGGCCTCCGGGCGGGCATGACAAGAATGTTAGAGGCT
CAGACAGACGGATAGTATGGTTTTCTGGTACACCAGGCGATAGAATTAAAGATCTATCAT
TGCGGGTGCCCCAGTCAGCGCGAGGTGCAGGGAGCAGCCGAGCTTCGAATTGGCACCTTC
CTATTATAACCGCCCCTCCGGGAGAGCCGGGTTCCGAGTACAGCTTGTCTTAACAGAAAG
GCATGGAAGACGAGGTCCACCCAGCCTTCGGGGTTCGGCATCAAAGTCGGCAACAGAGTA
CAGCCCAACAGGGGGAAGATTTACACGCGTACACCTTATATACCACCTTGCTGTCGCGGA
CTAATCTA
