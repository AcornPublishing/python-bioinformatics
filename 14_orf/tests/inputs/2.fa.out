M
MAEGGYRTSNHGSSL
MAYPRQYVRCLPKW
MDARISTTFELPYLTICQAFTF
MDKHKKWTPVFLLPLSCPISLSVRHLHSDG
MGLQ
MIDL
MLEAQTDG
MLWTNTRNGRPYFYYL
MPARRPRFLLTQQERRRPAFARLLVVCQTVSLSSGETVECDSG
MPCYGQTQEMDARISTTFELPYLTICQAFTF
MPDGFFVQWGNG
MPDR
MPFC
MPNPEGWVDLVFHAFLLRQAVLGTRLSRRGGYNRKVPIRSSAAPCTSR
MRLSRIALNRFPTGQRNRLAYDK
MSEETD
MTRMLEAQTDG
MTTQKAGIQWAYNKLLTTAFGVTKLD
MVFWYTRR
